examples/tests/sdk_tests/std/range.test.w

```diff
@@ -1,2 +1,52 @@
 //-----------------------------------------------------------------------------
-// TODO: https://github.com/winglang/wing/issues/2785
\ No newline at end of file
+// rangeOf
+
+assert(std.Range.of(1,5,true).join() == "1,2,3,4,5");
+
+test "rangeOf" {
+    assert(std.Range.of(1,5,true).join() == "1,2,3,4,5");
+}
+
+assert(std.Range.of(1,5,false).join() == "1,2,3,4");
+
+test "rangeOfWithoutLastNumber" {
+   assert(std.Range.of(1,5,false).join() == "1,2,3,4");
+}
+
+assert(std.Range.of(1,5).join() == "1,2,3,4");
+
+test "rangeOfWithoutLastNumberDefault" {
+   assert(std.Range.of(1,5).join() == "1,2,3,4");
```

Comment:
perhaps an `assert(std.Range.of(x, y).join() == std.Range.of(x, y, false).join());` too? this could confirm that the default boolean does not get changed accidentally.